package.json

```diff
@@ -61,10 +61,12 @@
     "ldapjs": "1.0.0"
   },
   "devDependencies": {
+    "babel-preset-es2015": "^6.14.0",
     "chai": "3.5.0",
     "eslint": "2.11.1",
     "font-awesome": "4.6.3",
     "grunt": "1.0.1",
+    "grunt-babel": "^6.0.0",
```

Comment:
Any chance we could live without `grunt-babel` or other grunt stuff? Our recent changes managed to avoid using Grunt, and frankly I'm hoping we can just get rid of it soon (and just use npm scripts or JS scripts if things are a bit too complex for npm scripts).